modular_skyrat/master_files/code/modules/clothing/suits/skinsuits.dm

```diff
@@ -0,0 +1,98 @@
+		and for what remains to stay in the frame. These are typically made of up to four layers of the stuff, working unintentionally to ensure the faces of their wearers are difficult to identify. \n\n\
+		In addition, the interlayer gives the helms sound insulation properties, and the ability to block UV radiation. \
+		These helmets are known for being distinctly uncomfortable in comparison to their civilian counterparts; cramped and with only the most barebones climate control tech.\n\n\ "
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/head/akula.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head/akula.dmi'
+	icon_state = "skinsuithelmet"
+
+/obj/item/clothing/head/helmet/space/skinsuit_helmet/equipped(mob/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_ICLOTHING)
+		return
+	apply_wetsuit_status_effect(user)
+
+/obj/item/clothing/head/helmet/space/skinsuit_helmet/dropped(mob/user)
+	. = ..()
+	remove_wetsuit_status_effect(user)
+
+/obj/item/clothing/head/helmet/space/skinsuit_helmet/Destroy()
+	. = ..()
+	var/mob/user = loc
+	if(!istype(user))
+		return
+	remove_wetsuit_status_effect(user)
+
+/obj/item/clothing/suit/armor/riot/skinsuit_armor
+	name = "Shorebreaker plating"
+	desc = "'Shorebreaker'-pattern Stardress plating was developed for Azulean boarding teams in service of the New Principalities. \
+		Kept resilient and maneuverable, Shorebreaker armor is built predominantly close-quarters breaching. \n\n\
+		Having sparse plating around the wearer's arms and legs, boarding teams are encouraged after their pod lands to do one thing: \
+		make use of their rapid movement, and render their enemies past tense. The armor is relatively hardy against ranged weapons, \
+		but the alloys involved are primarily constructed around resisting strikes from boarding axes, lances, and other common pirate-repellant weapons.\n\n\ "
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits/akula.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suits/akula.dmi'
+	icon_state = "skinsuitarmor"
+	base_icon_state = "skinsuitarmor"
+
+
+/obj/item/clothing/suit/armor/riot/skinsuit_armor/equipped(mob/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_ICLOTHING)
+		return
+	check_tail(user)
+	update_appearance()
+
+/obj/item/clothing/suit/armor/riot/skinsuit_armor/dropped(mob/user)
+	. = ..()
+	check_tail(user)
+	update_appearance()
+
+/obj/item/clothing/suit/armor/riot/skinsuit_armor/Destroy()
+	. = ..()
+	var/mob/user = loc
+	if(!istype(user))
+		return
+	remove_wetsuit_status_effect(user)
+
+// Pick an icon_state that matches nicer with tails if one is found on the wearer
+/obj/item/clothing/suit/armor/riot/skinsuit_armor/proc/check_tail(mob/living/carbon/human/user)
+	icon_state = base_icon_state
+	if(!user.dna.species.mutant_bodyparts["tail"])
+		return
+	icon_state = "skinsuitarmor_cutback"
```

Comment:
What about tails that are smaller, like, say, a Felinid's tail? What does it look like then?

Reply:
It just means they don't have a butt-plate.

Reply:
Fair enough I guess.